src/msg/async/frames_v2.h

```diff
@@ -129,27 +129,20 @@ static_assert(std::is_standard_layout<preamble_block_t>::value);
 // In addition to integrity/authenticity data each variant of epilogue
 // conveys late_flags. The initial user of this field will be the late
 // frame abortion facility.
-struct epilogue_plain_block_t {
+struct epilogue_crc_r0_block_t {
   __u8 late_flags;
   ceph_le32 crc_values[MAX_NUM_SEGMENTS];
 } __attribute__((packed));
-static_assert(std::is_standard_layout<epilogue_plain_block_t>::value);
+static_assert(std::is_standard_layout<epilogue_crc_r0_block_t>::value);
 
-struct epilogue_secure_block_t {
+struct epilogue_secure_r0_block_t {
   __u8 late_flags;
   __u8 padding[CRYPTO_BLOCK_SIZE - sizeof(late_flags)];
 
   __u8 ciphers_private_data[];
 } __attribute__((packed));
-static_assert(sizeof(epilogue_secure_block_t) % CRYPTO_BLOCK_SIZE == 0);
-static_assert(std::is_standard_layout<epilogue_secure_block_t>::value);
-
-
-static constexpr uint32_t FRAME_PREAMBLE_SIZE = sizeof(preamble_block_t);
-static constexpr uint32_t FRAME_PLAIN_EPILOGUE_SIZE =
-    sizeof(epilogue_plain_block_t);
-static constexpr uint32_t FRAME_SECURE_EPILOGUE_SIZE =
-    sizeof(epilogue_secure_block_t);
+static_assert(sizeof(epilogue_secure_r0_block_t) % CRYPTO_BLOCK_SIZE == 0);
+static_assert(std::is_standard_layout<epilogue_secure_r0_block_t>::value);
```

Comment:
Let's use this occasion to switch to C++17's `std::is_standard_layout_v`.

Reply:
Will do.